server/protocols/jmap/src/main/java/org/apache/james/jmap/model/MessageContentExtractor.java

```diff
@@ -84,6 +104,11 @@ private MessageContent parseMultipartAlternative(Multipart multipart) throws IOE
         return new MessageContent(textBody, htmlBody);
     }
 
+    private MessageContent parseMultipartRelated(Multipart multipart) throws IOException {
+        Optional<String> htmlBody = getFirstMatchingTextBody(multipart, "text/html");
```

Comment:
Interesting question. Do you really think it is possible to have multipart/related without html?